qiita_pet/templates/workflows.html

```diff
@@ -55,14 +47,25 @@
   return title;
 }
 
+function makePopper(node) {
+  node.tippy = tippy(node.popperRef(), {
+    content: () => {
+      let content = document.createElement('div');
+      content.innerHTML = node.data('tooltip');
+      return content;
+    },
+    trigger: 'manual' // probably want manual mode
```

Comment:
What would non-manual mode be?

Reply:
😄 😄 😄    that's from the [example for tippy/popper](https://github.com/cytoscape/cytoscape.js-popper#usage-with-tippyjs) and looking closer there are [2 other methods](https://atomiks.github.io/tippyjs/#triggers) - however, not manual but it works ...

Reply:
ah okay :)